src/simulation/dynamics/reactionWheels/reactionWheelStateEffector.cpp

```diff
@@ -44,10 +46,17 @@ ReactionWheelStateEffector::ReactionWheelStateEffector()
 
 ReactionWheelStateEffector::~ReactionWheelStateEffector()
 {
-    for (long unsigned int c=0; c<this->rwOutMsgs.size(); c++) {
-        free(this->rwOutMsgs.at(c));
+    // Clear output messages vector
+    for (unsigned int i = 0; i < rwOutMsgs.size(); i++) {
```

Comment:
class variables should use `self.rwOutMsgs`.  Please correct throughout the file.